src/main/java/reposense/git/GitLog.java

```diff
@@ -45,7 +45,7 @@ public static String getWithFiles(RepoConfiguration config, Author author) {
         command += GitUtil.convertToGitDateRangeArgs(config.getSinceDate(), config.getUntilDate());
         command += " --pretty=format:" + PRETTY_FORMAT_STRING + " --stat";
         command += GitUtil.convertToFilterAuthorArgs(author);
-        command += GitUtil.convertToGitFormatsArgs(config.getFormats());
+        command += GitUtil.convertToGitFormatsArgs(config.getFileTypeManager());
```

Comment:
should use `config.getFileTypeManager().getFormats()` instead, principle of least knowledge